cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-higgsfourleptons.json

```diff
@@ -0,0 +1,92 @@
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Wunsch, Stefan",
+        "orcid": "0000-0002-4485-2972"
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_published": "2021",
+    "distribution": {
+      "formats": [
+        "cc",
+        "py",
+        "zip"
+      ],
+      "number_files": 4,
+      "size": 999
+    },
+    "doi": "10.7483/OPENDATA.CMS.TODO",
+    "experiment": "CMS",
+    "files": [
+      {
+        "checksum": "adler32:999",
+        "size": 999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/TODO/HiggsToFourLeptonsNanoAODOutreachAnalysis-1.0.zip"
+      }
+    ],
+    "license": {
+      "attribution": "GNU General Public License (GPL) version 3"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "12360",
+    "run_period": [
+      "Run2012A",
+      "Run2012B"
+    ],
+    "source_code_repository": {
+      "url": "www.github.com/cms-opendata-analyses/HiggsToFourLeptonsNanoAODOutreachAnalysis"
```

Comment:
Could the name of the repository be shortened to HiggsTo4LeptonsNanoAODOutreachAnalysis?

Reply:
I'd rather keep it as it is, otherwise the git repo has to be renamed :/

Reply:
Ok, no problem